### PR TITLE
add new option lowerCaseSmallWords

### PR DESCRIPTION
**Pull request by bas-go1**

### CODE REVIEW

File: packages/title-case/src/index.ts [hunk 1]
```diff
@@ -76,6 +77,7 @@ export function titleCase(
     titleTerminators = TITLE_TERMINATORS,
     smallWords = SMALL_WORDS,
     wordSeparators = WORD_SEPARATORS,
+    lowerCaseSmallWords = false,
```
Comment (blakeembrey) [inline packages/title-case/src/index.ts:80]: Should this lower case at the end or beginning of a sentence?

**Review by blakeembrey — COMMENTED** [review]
Can you help me understand why we want this feature? I understand what you're doing now, but I'm still not sure why. That would help me since I need to maintain it, and it's not obvious to me.

File: packages/title-case/src/index.ts (hunk 1 above)
Comment (bas-go1) [reply packages/title-case/src/index.ts:80]: Hi Blake, so this is for the case where the input contains small words that are not in lowercase but the current implementation does not lowercase them. Regard your second question, I think it should not lowercase the beginning and ending of a sentence.

**Comment by blakeembrey** [comment]
I'm going to close this out as won't merge for now, it's not a feature I'm comfortable maintaining in the long run.

**Pull request closed without merging**